src/objects/avatar/Avatar.ts

```diff
@@ -216,6 +218,16 @@ export class Avatar extends RoomObject implements IMoveable, IScreenPositioned {
 
   set direction(value) {
     this._direction = value;
+    this._headDirection = value;
```

Comment:
I'm not quite sure if we want the behavior, that when we change the direction it changes the headDirection aswell.

I would have expected, once I set the headDirection, it will stay with that headDirection until I set it to something else, or I set it to undefined again.